README.md

```diff
@@ -1,31 +1,54 @@
 ## About
 
-Code duplication is a big issue. As developers move from from project to project we often copy and paste snippets of code between projects. While this is quick and easy its very hard to maintain.
-
-This project is intended to act as a place where NodeJS developers can store and share useful snippets of code.
+For applications that need to manage AirWatch, we provide an API wrapper of the AirWatch Client
 
 ## Usage
 
 Install via `npm` as you would with any other package by supplying both the repo URL and branch or tag reference. In the example below the tag _v0.0.1_ is used.
 
 ```console
-npm i -S @bcgov/common-nodejs-utils
+npm i -S @bcgov/mdms-nodejs-client
 ```
 
 You can also directly add it to your `package.json` file by inserting the following line into your dependencies section:
 
 ```json
-"@bcgov/common-nodejs-utils": "git+https://git@github.com/bcgov/common-nodejs-utils.git#v0.0.1",
+"@bcgov/mdms-nodejs-client": "git+https://git@github.com/bcgov/mdms-nodejs-client.git#v0.1.0",
 ```
 
 Once you installed you can can use it like any other module:
 
 ```javascript
-import { errorWithCode } from '@bcgov/common-nodejs-utils';
+import { getAllOrgGroups } from '@bcgov/mdms-nodejs-client';
 ```
 
 \* You can also use the ssh protocol preferred.
 
+For Example, to get a list of all organization groups in AirWatch, including the sub origanization groups:
+
+```javascript
+// Set the AW credentials as env vars:
+//  AW_HOST
+//  AW_TENANT_CODE
+//  AW_USERNAME
+//  AW_PASSWORD
+
+const awCredentials = {
```

Comment:
I'd just call this `credentials` since there is only one set set of credentials.